### DISCUSSION TITLE
Port forward coder workspace + traefik

### DISCUSSION THREAD

**Discussion by hoangndst** [coder/coder#6061] [General] [answered]
I deployed the code with docker and used Traefik to reverse proxy using the domain `https://code.hoang***.f***.org`. Currently, I have the problem of not being able to access the workspace service when port forwarding. For example, I create a workspace and run React on port 3000. The link would be `https://3000--main--port-test--hoangndst.code.hoang***.f***.org`. But it's not accessible.
How to access it?

**Comment by matifali** [comment]
Coder requires your reverse proxy to forward WebSockets too. Please check other examples to see how it is done there. https://coder.com/docs/v2/latest/admin/configure#tls--reverse-proxy

**Comment by renesas-brandon-hussey** [comment, marked as answer]
@hoangndst did you find a solution? I'm using Traefik and Coder in docker and this seemed to work for me:

```yml
    labels:
      - "traefik.enable=true"
      - "traefik.http.routers.coder.rule=HostRegexp(`coder.<hostname>`,`{subdomain:.*}.coder.<hostname>`)"
      - "traefik.http.routers.coder.entrypoints=web"
      - "traefik.http.services.coder.loadbalancer.server.port=7080"
```

I'm running into some other issues though. I'm not sure if they're related to Traefik or not.
